src/devices/device_models/AC_branches.jl

```diff
@@ -26,89 +26,252 @@ function flow_variables(ps_m::CanonicalModel,
                         devices::PSY.FlattenedVectorsIterator{B},
                         time_steps::UnitRange{Int64}) where {B <: PSY.ACBranch,
                                                              S <: StandardPTDFForm}
+                                                         
+    #Get PowerModels dicts
+    var_name = Symbol("Fbr_$(B)")
+
+    ps_m.variables[var_name] = PSI._container_spec(ps_m.JuMPmodel,
+                                                   (d.name for d in devices),
+                                                    time_steps)
+
+    for (ix, d) in enumerate(devices)
+        bus_fr = d.connectionpoints.from.number
+        bus_to = d.connectionpoints.to.number
+        for t in time_steps
+            
+            #Active Power Variables
+            ps_m.variables[var_name][d.name,t] = JuMP.@variable(ps_m.JuMPmodel,
+                                                                base_name="$(bus_fr),$(bus_to)_{$(d.name),$(t)}",
+                                                                upper_bound = d.rate,
+                                                                lower_bound = -d.rate,
+                                                                 start = 0.0)
+        end
 
-    add_variable(ps_m,
-                 devices,
-                 time_steps,
-                 Symbol("Fbr_$(B)"),
-                 false)
+    end
 
     return
 
 end
 
+function flow_variables(ps_m::CanonicalModel,
+                        system_formulation::Type{S},
+                        devices::PSY.FlattenedVectorsIterator{B},
+                        time_steps::UnitRange{Int64}) where {B <: PSY.ACBranch,
+                                                             S <: PM.AbstractPowerFormulation}
+    #Get PowerModels dicts
+    pm_object = ps_m.pm_model
+    ref = pm_object.ref
+    var = pm_object.var
```

Comment:
Instead of accessing the internal ref and var objects.  I recomend to use the ref and var functions (e.g. `var(pm_object, t, :p)`).  These allow you to ignore more advanced PowerModels features, like conductors.

Reply:
solved via b9d9919